core/src/main/java/edu/wpi/grip/core/operations/composite/AdvancedFilterContoursOperation.java

```diff
@@ -165,19 +190,41 @@ public void perform() {
     for (int i = 0; i < inputContours.size(); i++) {
       final Mat contour = inputContours.get(i);
 
-      final Rect bb = boundingRect(contour);
-      if (bb.width() < minWidth || bb.width() > maxWidth) {
+      double width;
+      double height;
+      if (rotatedRect) {
+        final opencv_core.RotatedRect bb = minAreaRect(contour);
+        opencv_core.Point2f points = new opencv_core.Point2f(4);
```

Comment:
`RotatedRect` and `Point2f` should be imported directly from `org.bytedeco.javacpp.opencv_core` to avoid the `opencv_core.RotatedRect` crap